Makefile

```diff
@@ -278,6 +282,12 @@ react-build:
 	rsync -av --delete --delete-excluded --exclude=reports \
 		frontend/build/ lib/streamlit/static/
 
+.PHONY: react-build-fast
+react-build-fast:
+	cd frontend/ ; yarn run buildFast
+	rsync -av --delete --delete-excluded --exclude=reports \
+		frontend/build/ lib/streamlit/static/
+
```

Comment:
Since `react-build-fast` is only used by `frontend-fast`, I don't think we gain anything by having it be a separate make rule. Can we remove `react-build-fast` and just stick its logic directly into `frontend-fast`?

Reply:
Yeah, I agree I've removed alias logic from the PR 👍